week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -16,7 +18,10 @@ function convertToUSD() {}
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  let UKMoney = price * 5.7;
+  return UKMoney + (0.01 * UKMoney);
```

Comment:
Good job! How can you make this simpler so that you only use `UKMoney` once in this statement? :)

Reply:
instead line 22 and 23 :
return ( ( price * 5.7 ) + ( 0.01 * ( price * 5.7 ) );